runner.py

```diff
@@ -1140,8 +1140,13 @@ def read_container_logs(self):
 
             if log.stdout:
                 self.add_to_log(container_id, f"stdout: {log.stdout}")
-                if container_info['read-sci-stdout']:
-                    for line in log.stdout.splitlines():
+
+                for line in log.stdout.splitlines():
```

Comment:
I would probably check beforehand if we really need to loop over the logs. The problem ist that in some cases the logs become quite big and if we don't have `read-notes` or `read-sci` enabled we don't really need to do that.